### PR TITLE
man.vim: Ignore $MANWIDTH and use vim's soft wrap

### LINKED ISSUE #9017: Man.vim: mentioned manpages are not detected correctly when they span over several lines
- `nvim --version`: 
```
NVIM v0.3.1
Build type: Release
LuaJIT 2.0.5
Compilation: /usr/bin/cc -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -Wconversion -O2 -DNDEBUG -DMIN_LOG_LEVEL=3 -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wimplicit-fallthrough -Wvla -fstack-protector-strong -fdiagnostics-color=auto -Wno-array-bounds -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -I/build/neovim/src/build/config -I/build/neovim/src/neovim-0.3.1/src -I/usr/include -I/build/neovim/src/build/src/nvim/auto -I/build/neovim/src/build/include
Compiled by builduser

Features: +acl +iconv +jemalloc +tui 
See ":help feature-compile"

   system vimrc file: "$VIM/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```
- Vim (version: 8.1) behaves differently? No
- Operating system/version: Arch Linux
- Terminal name/version: Urxvt
- `$TERM`: `tmux-256color`

### Steps to reproduce using `nvim -u NORC`

```
nvim -u NORC
:Man 2 fork " Just an example, not all manpages show this behaviour
```

### Actual behaviour

'Links' to other man pages at the `SEE ALSO` section at the bottom of the page which span over several lines are not detected by the plugin. This affects not only the syntax these words get but also the possibility to jump to those manpages when they are under the cursor - with `CTRL-]`/`K`.

### Expected behaviour

Every manpage 'Link' is detected and available for jumping to it with `CTRL-]`/`K`.

### Solution suggestion

What if we'll use `wrap` and `linebreak` with `breakindent` instead of creating hard wraps when the man page is read? This would have an additional benefit - resizing the window where there's the man page won't require reloading the man page in order to fix the fake (hard) wrappings.

### PR DESCRIPTION
**Pull request by doronbehar**
Fixes #9017

### CODE REVIEW

File: runtime/ftplugin/man.vim [hunk 1]
```diff
@@ -48,5 +48,17 @@ if get(g:, 'ft_man_folding_enable', 0)
   setlocal foldnestmax=1
 endif
 
+if !get(g:, 'ft_man_wrap_disable', 0)
+  setlocal wrap
+endif
+
+if ! get(g:, 'ft_man_linebreak_disable', 0)
+  setlocal wrap
```
Comment (justinmk) [inline runtime/ftplugin/man.vim]: what 'linebreak' setting is needed, if any?

**Comment by justinmk** [comment]
I noticed a problem and pushed an update. See the last commit.

**Comment by e-kwsm** [comment]
dc256e3 made the cursor go to the bottom of the manpage.
A simple workaround is to add `exe 'normal! gg'`.

**Comment by justinmk** [comment]
@e-kwsm fixed by a1976c739097

**Pull request closed without merging**